### ISSUE TITLE
some questions about parameters for training

### ISSUE THREAD

**Issue by ShireLiuSZ** [yulunzhang/RNAN#15]
Hi! Thanks for your wonderful work and open sources. 
I'd like to reproduce the result of denoising. Here are some questions to consult:
1. What dataset is used for validation and how many iamges are used? (The readme mentioned that the validation set is div2k_val100, but in the options.py in the open source code, the default parameter shows that only  the first 5 images of div2k_val_100 0801 ～ 0805 as used in the validation set)

2. What is the number of training epoches in the grayscale denoising task? How many days does the training take? (The parameter in options.py defaults to 1300 epoch, considering that the train set in each epoch is repeated 20 times, and the actual number of epochs is 26,000. We found that training 1/1300 epoch takes about 2 hours)

3. How to obtain the model corresponding the performance in paper? Did the test result shown in the paper use the best model on validation, or the model obtained from the last epoch?

4. Are the other parameters of the experiment to reproduce the result in paper the same as those given by options.py?

Looking forward to your apply. Thanks a lot!

Regards
